### ISSUE TITLE
apiSampleRequest 请求报错：Error 0: SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': '' is not a valid HTTP method. undefined

### ISSUE THREAD

**Issue by shupan** [apidoc/apidoc#787]
![image](https://user-images.githubusercontent.com/9160070/58492422-78550180-81a3-11e9-8b15-f8761cec0301.png)

Error 0: SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': '' is not a valid HTTP method.
undefined

**Comment by eyedean** [comment]
@shupan I ran into the same issue and found that it's trying to make an AJAX call from FILE protocol (when you are opening local index.html file in your browser) to HTTP protocol, where your server is running.

To fix that I set my Node.js server to serve the APIDoc as a static path when I am in dev mode.  That way, the URL to apidoc would be http://localhost:3000/apidoc for me and any request to http://localhost:3000/api/v1/getuser will be allowed! :)

Good luck.
